### DISCUSSION TITLE
Mapper: Show room names on map

### DISCUSSION THREAD

**Discussion by buhtz** [scandum/tintin#179] [General]
I do follow your docu do display the current map in a split window.

```
#split 16 1
#map flag vtmap on
```

Also do set room names on some of them via `#map set roomname Bar`. But that names do not appear on the map as you can see here:

![image](https://user-images.githubusercontent.com/11861496/230489463-0c817de8-d758-4aae-9bd0-c69bd6c84e5c.png)

Can I display them on the map somehow?

**Comment by scandum** [comment]
You can't. One thing you can do is set the room symbol using: `#map set roomsymbol <value>`, which can be between 1 and 5 characters long.

To get the names of nearby rooms you can use for example `#map list {distance}{15}`. Keep in mind the distance is based on the room weight, which by default increases by 1 for each exit, and 1 for each room.

**Reply by buhtz** [reply]
Is there something like a legend? The map is useless when I can't label the rooms somehow.

**Reply by fal3nsh** [reply]
Far from useless actually. You can give descriptions, names, notes, even symbols (as said) to any rooms. Use symbols for "shops", for example, and then you won't need any label. A label can still be set, tough : giving a "name" to room allow you to quick move the map to the room (using map commands / a good and quick alias will help), and even to "run" to them (speewalking). 

This kind of open mapper is the best thing I have come accross for mudding ! Be sur to study the help files, and enjoy :)

**Reply by dflan83** [reply]
Late to the thread but in a lot of the rooms that I am interested in, I will set a roomnote as poi (point of interest) or something similar. You can then depending on how your window is split write a script that will show these nearby room names depending on how far away that you are from them.
